spec/lib/registries/topic_taxonomy_registry_spec.rb

```diff
@@ -0,0 +1,94 @@
+require "securerandom"
+require 'spec_helper'
+require "gds_api/test_helpers/content_store"
+
+RSpec.describe Registries::TopicTaxonomyRegistry do
+  include ::GdsApi::TestHelpers::ContentStore
+
+  let(:content_id_one) { "top-level-taxon-one" }
+  let(:content_id_two) { "top-level-taxon-two" }
+  let(:top_level_taxon_one) { top_level_taxon(content_id_one) }
+  let(:top_level_taxon_two) { top_level_taxon(content_id_two) }
+
+  describe "when topic taxonomy API is unavailable" do
+    it "will return an (uncached) empty hash" do
+      clear_cache
+      topic_taxonomy_api_is_unavailable
```

Comment:
Could use `content_store_isnt_available` from the gds-api-adapters content store helper: https://github.com/alphagov/gds-api-adapters/blob/1dadd526f523a744413e166f42c4cd3f01e3084b/lib/gds_api/test_helpers/content_store.rb#L81